lib/bemxjst/match.js

```diff
@@ -48,7 +48,7 @@ function MatchCustom(template, pred) {
 }
 
 MatchCustom.prototype.exec = function exec(context) {
-  return this.body.call(context);
+  return this.body.call(context, context);
```

Comment:
Yeah, I know that. Still I have a small hope to avoid code compilation step, since destructuring assignment hasn't shipped to node.js yet.

Anyways both variants are good and I vote for this feature.